### PR TITLE
Test - finish writing tests

### PR DESCRIPTION
**Pull request by rise-erpelding**
# Description

This PR finishes creating tests related to the creation of our color grid! What's a color grid? It's a grid in which the colors gradually change from one side of the grid to the other. If you need a visualization of what that looks like, check [this](https://codepen.io/riseerpelding/pen/qBqzwvq) out!

A few tests already existed prior to creating this branch. Here we seek to add a few more to ensure that each of the functions that contribute to creating our color grid have, at a minimum, one test.

Here's some other background if that is helpful:

The code is organized into three core functions that that generate our grid: `makeGrid`, `makeRow`, and `makeStops`, plus utility functions (in the `color-utils` directory):
- `makeGrid` places the colors into a grid from one side to the other.
- `makeRow` figures out how to take two colors and generate all the intermediate colors between them.
- `convertHexToRGB` is pretty self-explanatory, it takes a hex color and turns it into and rgb color to take advantage of its numbers
- `convertRGBtoHex` is likewise self-explanatory, it converts the rgb color back to a hex color once we're done using the numbers
- `componentToHex` is a helper function for `convertRGBtoHex` that takes each number in the rgb color and turns it into a two-character hex value so that the three values together create the full hex color.
- `makeStops` compares two individual numbers from an rgb color (two r's, two g's, or two b's) and generates all of the numbers that should go in between to make the values as evenly spaced as possible.

## Spec
See [FSA2021-98](https://sparkbox.atlassian.net/browse/FSA2021-98)

## To Validate
1. Pull down the branch.
2. Run `npm test` to verify that all tests pass.
3. Check out the tests in `index.test.js`, `grid.test.js`, `row.test.js`, and `stops.test.js`. Do the tests actually test what they're supposed to be testing? Do you see anything potentially missing? Or anything extraneous? Is anything out of scope?

### CODE REVIEW

File: src/color-utils/index.js [hunk 1]
```diff
@@ -1,4 +1,7 @@
 const checkRGBValidity = (num) => {
+  if (typeof num !== 'number') {
+    throw new TypeError('num must be a number')
+  }
```
Comment (lahernandezb) [inline src/color-utils/index.js:4]: Nice job considering this edge case!

File: src/color-utils/index.test.js [hunk 2]
```diff
@@ -20,6 +20,12 @@ describe('checkRGBValidity function', () => {
   test('returns false for a number greater than 255', () => {
     expect(checkRGBValidity(260)).toBeFalsy();
   });
+
+  test('throws an error if an empty string is passed in', () => {
+    expect(() => {
+      checkRGBValidity('');
+    }).toThrow('num must be a number');
+  });
```
Comment (lahernandezb) [inline src/color-utils/index.test.js:28]: 🎉
